src/main/kotlin/org/jetbrains/research/transformations/anonimization/Util.kt

```diff
@@ -0,0 +1,35 @@
+package org.jetbrains.research.transformations.anonimization
+
+import com.github.gumtreediff.tree.ITree
+
+interface IdCounter {
+    val currentId: Int
+}
+
+data class AnonymousConfig(
+        override var currentId: Int = 0,
+        // old_label -> list[(new_label, parent_prefix)]
+        val namesMap: MutableMap<String, List<Pair<String, String>>> = mutableMapOf()
```

Comment:
Could you please add a comment and clarify what is a namesMap? Why it stores a list of pairs of strings?

Reply:
let me show my mind flow ion this example:

```
def fun1 (a):
   x = x + 1
   return x

def fun2 (x):
   x = x + 2
   return x

x = 0
print(fun1(x), fun2(x))
```

should be anonymized into 
```
def f1 (f1_a1):
   f1_v1 = f1_a1 + 1
   return f1_v1

def f2 (f2_a1):
   f2_v1 = f2_a1 + 2
   return f2_v1

v1 = 0
print(f1(v1), f2(v1))
```

Reply:
You have to keep in memory, that var x can be either f1_v1, f1_a1, f2_v1, f2_a1 or  v1. And it is more convenient to store var and prefix, then concatenated version. because I choose a name for new var/arg, comparing prefixes.

Reply:
so I have a list:
x -> [ (v1, f1), (a1, f1), (v1, f2), (a1, f2), (v1, "")]

Reply:
Sorry, but shouldn't it be like this after anonymization?
```
def f1 (f1_a1):
   f1_a1 = f1_a1 + 1
   return f1_a1
```

Also, there can be nested functions in python, for example, 
```
def first_function():
    def second_feunction():
        b = 5
        return b
    a = second_function()
    return a
```

so should `b` after anonymization have prefixes both from first_function and second_function? If so, you cannot store prefixes in pairs, I suppose...

Reply:
I fixed it according to @elena-lyulina  examples